components/automate-cli/pkg/verifyserver/server/api/v1/softwareversion.go

```diff
@@ -0,0 +1,19 @@
+package v1
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) CheckSoftwareVersion(c *fiber.Ctx) {
+	services, err := h.SoftwareVersionService.GetSoftwareVersionServices(c.Query("node_type"))
+	if err != nil {
+		c.Status(fiber.StatusBadRequest).JSON(response.BuildFailedResponse(&fiber.Error{Message: err.Error(),Code: 400}))
```

Comment:
send 400 only if the request query or body is bad : use it like this `c.Next(&fiber.Error{Message: err.Error(),Code: fiber.StatusBadRequest})`
for unknown server errors use fiber.StatusInternalServerError

Reply:
Done